runners/flink/job-server/flink_job_server.gradle

```diff
@@ -193,17 +193,17 @@ def portableValidatesRunnerTask(String name, boolean streaming, boolean checkpoi
         excludeCategories 'org.apache.beam.sdk.testing.UsesPerKeyOrderInBundle'
     },
     testFilter: {
-      // TODO(BEAM-10016)
+      // TODO(https://github.com/apache/beam/issues/20269)
       excludeTestsMatching 'org.apache.beam.sdk.transforms.FlattenTest.testFlattenWithDifferentInputAndOutputCoders2'
-      // TODO(BEAM-12039)
+      // TODO(https://github.com/apache/beam/issues/20843)
       excludeTestsMatching 'org.apache.beam.sdk.testing.TestStreamTest.testDiscardingMode'
-      // TODO(BEAM-12038)
+      // TODO(https://github.com/apache/beam/issues/20844)
       excludeTestsMatching 'org.apache.beam.sdk.testing.TestStreamTest.testLateDataAccumulating'
       // TODO(BEAM-12710)
       excludeTestsMatching 'org.apache.beam.sdk.testing.TestStreamTest.testFirstElementLate'
       // TODO(BEAM-13498)
       excludeTestsMatching 'org.apache.beam.sdk.transforms.ParDoTest$TimestampTests.testProcessElementSkew'
```

Comment:
Once this is done we should do another pass for references to closed jiras. Either the jira wasn't actually fixed and we should re-open the task, or the comment is out-dated and should be removed. In this case it looks like we may even be skipping a test that we shouldn't be.

Reply:
Yeah, I agree that would be good to do - that probably would need to be driven by SDK owners though (e.g. it would be hard for me to evaluate quickly whether a ticket should have been resolved in the Java SDK)